bulletroboy/operator.py

```diff
@@ -22,8 +22,14 @@ def __init__(self, body_id):
 		self.links = self.get_links()
 		self.movements = Movements(self)
 		self.ef_publisher = self.create_publisher(PoseStamped, '/roboy/simulation/operator/pose/endeffector', 10)
+		self.srv = self.create_service(HumanLinkIdFromName, 'human_link_id_from_link_name', self.human_link_id_from_link_name)
```

Comment:
as you can see in the line above, there's a naming convention for the services and topics. it goes like `/roboy/simulation/operator.....`